ci/baremetal/baremetal-cluster.lokocfg.envsubst

```diff
@@ -31,4 +31,8 @@ cluster "bare-metal" {
     "node2",
     "node3",
   ]
+  # Adds oidc flags to API server with default values.
```

Comment:
The commit message here is wrong, mentions `aws`.

I would also mention in the commit message, that it enables validation of the configuration for this platform.